packages/components/src/components/tab/tab.stories.ts

```diff
@@ -55,23 +55,76 @@ export const Default = {
 
 export const Variant = {
```

Comment:
For the variants when tab is open can you use the same copy on both default and container variants?
- Provident illo neque vel ex. Inventore perspiciatis delectus nisi doloremque soluta inventore.
- Lorem ipsum dolor sit amet, consectetur adipiscing elit.
- Inventore perspiciatis delectus nisi doloremque soluta inventore.

Reply:
Lets also apply this order to the rest of examples to reinforce that tabs as changing please

Reply:
Updated